templates/collection.nomad

```diff
@@ -92,9 +92,39 @@ job "collection-${name}" {
           pg = 5432
         }
       }
-      env {
-        POSTGRES_USER = "snoop"
-        POSTGRES_DATABASE = "snoop"
+      template {
+        data = <<EOF
+          POSTGRES_USER = "snoop"
+          POSTGRES_DATABASE = "snoop"
+          {{- with secret "liquid/collections/${name}/snoop.postgres" }}
+            POSTGRES_PASSWORD = {{.Data.secret_key}}
+          {{- end }}
+        EOF
+        destination = "local/postgres.env"
+        env = true
+      }
+      template {
+        data = <<-EOF
+        #!/bin/sh
+        set -ex
+        pwd
+        date
+        if grep -Fq "$host all all all trust" $PGDATA/pg_hba.conf
+        then
+          psql -U snoop -c "ALTER USER snoop password '$POSTGRES_PASSWORD'"
+          sed -i '$d' $PGDATA/pg_hba.conf
+          sed -i '$d' $PGDATA/pg_hba.conf
```

Comment:
What is the goal here with `sed`? And why twice?

Reply:
it deletes the last two lines of the `pg_hba.conf`. The follwing lines replace them.